libnmstate/nm/ovs.py

```diff
@@ -50,6 +53,20 @@ def has_ovs_capability():
     return nmclient.NM.Capability.OVS in nm_client.get_capabilities()
 
 
+def is_ovs_running():
+    try:
+        subprocess.run(
+            ("systemctl", "status", "openvswitch"),
+            stdout=subprocess.DEVNULL,
+            stderr=subprocess.DEVNULL,
+            check=True,
+            timeout=SYSTEMCTL_TIMEOUT_SECONDS,
+        )
+        return True
+    except subprocess.CalledProcessError:
```

Comment:
You have to handle the timeout exception also. I would suggest simple `try Exception`.

Reply:
Done, thanks!